Cargo.lock

```diff
@@ -1056,8 +1056,8 @@ dependencies = [
  "libc",
  "log",
  "once_cell",
- "socket2 0.5.7",
- "windows-sys 0.52.0",
+ "socket2",
+ "windows-sys 0.60.2",
```

Comment:
Unrelated? Also note that Firefox uses `v0.52.0`:

https://github.com/mozilla-firefox/firefox/blob/main/Cargo.lock#L8242-L8249